apps/challenges/models.py

```diff
@@ -73,6 +80,14 @@ def is_active(self):
         return False
 
 
+@receiver(signals.post_save, sender=Challenge)
+def create_post_challenge(sender, instance, created, **kwargs):
+    if instance._Challenge__original_evaluation_script is False:
+        print "First time evaluation script added !"
```

Comment:
Can we use `logger` here. I know these print statement will be removed in the future, but as of now lets try following practices and use logger.